### PR TITLE
Update install versions

### PR DESCRIPTION
**Pull request by martinRenou**
Making this draft for now.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -4,26 +4,24 @@ channels:
...
   - ipywidgets
   - bqplot
-  - ipyleaflet=0.12.6
+  - ipyleaflet
   - pythreejs
   - ipycanvas
-  - ipyevents
-  - ipysheet
+  # - ipyevents
+  # - ipysheet
```
Comment (martinRenou) [inline environment.yml:18]: We should maybe remove this?

**Comment by mwcraig** [comment]
This looks good to me so far. I'll spend some time this week looking at a potential astro image viewer to mention during the tools section. 

We don't have anything that requires ipyevents, though, so we can drop that.

File: install_check.py [hunk 2]
```diff
@@ -3,7 +3,8 @@
 from subprocess import check_call, CalledProcessError
 
 
-FIX_PREFIX = '----->'
+SUCCESS_PREFIX = '🎉'
```
Comment (martinRenou) [inline install_check.py:6]: 💯 👍🏼

**Comment by mwcraig** [comment]
I've pinned all of the widget packages I think. @martinRenou the ipyvolume lab extension now installs but I still just see white boxes when I run in lab.

@ibdafna @marimeireles @bsyouness @martinRenou  -- I'll merge this in about an hour if there are no objections.

**Pull request merged**